_layouts/default.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
Might want the `<meta http-equiv="X-UA-Compatible" content="IE=edge"/>` header for BC Gov employees using their default IE11 (IE7 compatibility mode) settings. Unless that's provided somewhere by default on Github pages

Reply:
good idea - in commit.